packages/node/src/lib/metrics-log.ts

```diff
@@ -34,21 +34,23 @@ export interface GroupingObject {
 export interface OutgoingLogBody {
   _id?: string;
   _version: number;
-  clientIPAddress: string;
+  clientIPAddress?: string;
```

Comment:
Are we expecting this to be null occasionally? The value should always be here, right?

Reply:
updated this type in https://github.com/readmeio/metrics-sdks/pull/900/commits/af9ce2a12d111a6d280978bb5f452a795dbb70ac, what do you think? this is happening because `req.socket.remoteAddress` can be undefined on this line:

https://github.com/readmeio/metrics-sdks/blob/af9ce2a12d111a6d280978bb5f452a795dbb70ac/packages/node/src/lib/construct-payload.ts#L151